tests/tasks/test_vendor.py

```diff
@@ -20,10 +20,10 @@ async def test_run_should_set_enterprise_id(self, snmpsim, snmp_test_port):
         # The "public" test fixture is from an HP switch
         assert state.devices[device.name].enterprise_id == 11
 
-    @patch("zino.state.devices", DeviceStates())
+    @patch("zino.state.state", ZinoState())
```

Comment:
This is the type of silly patching that will become obsolete if we implement #42